src/collective/listingviews/browser/views/base.py

```diff
@@ -44,7 +43,8 @@ def __init__(self, context, request):
         self.filters = dict(
             localshort = lambda item, value: plone_util.toLocalizedTime(value, long_format=0),
             locallong = lambda item, value: plone_util.toLocalizedTime(value, long_format=1),
-            tolink = lambda item, value: '<a href="%s">%s</a>'%(item.getURL(), value)
+            tolink = lambda item, value: '<a href="%s">%s</a>'%(item.getURL(), value),
+            toImage = lambda item, value: '<img src="%s"/>' % ( item.getURL() if item.portal_type =='Image' else item.getURL() +'/@@images/image' if item.portal_type =='News Item' else '')
```

Comment:
I think this is cheating as you don't use 'value'. so you can't use this for actual image fields. 
like why can't you do 'image:tag_mini' or 'my_image:tag_large'?

Reply:
ie the idea of a filter is that it should be able to act on other custom fields. so if I make a custom field that returns the url of my special field, then I should be able to use a filter to then turn that into a image tag.